src/ggml-metal/ggml-metal.m

```diff
@@ -1258,7 +1260,7 @@ static bool ggml_metal_supports_op(const struct ggml_backend_metal_device_contex
         case GGML_OP_TIMESTEP_EMBEDDING:
         case GGML_OP_ARGSORT:
         case GGML_OP_LEAKY_RELU:
-            return true;
+            return op->src[0]->type == GGML_TYPE_F32;
```

Comment:
This caused the Metal backend to crash because `op->src[0]` is NULL (`ggml_arange()` does not have source tensors).

Reply:
Thanks, fixed!